x-pack/plugins/apm/common/sort_direction_rt.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import * as t from 'io-ts';
+
+export const toSortDirectionRt = t.union([t.literal('asc'), t.literal('desc')]);
+export type SortDirection = t.TypeOf<typeof toSortDirectionRt>;
```

Comment:
can you make this an enum (and use it in the literal)? This means we can also use it as a value.